modules/ank/arrow-ank/src/main/kotlin/arrow/ank/ank.kt

```diff
@@ -24,9 +40,9 @@ fun <F> Concurrent<F>.ank(source: Path, target: Path, compilerArgs: List<String>
     !effect { printConsole("Current heap used: ${(heapSize - Runtime.getRuntime().freeMemory()).humanBytes()}") }
     !effect { printConsole("Starting ank with Heap Size: ${heapSize.humanBytes()}, Max Heap Size: ${heapMaxSize.humanBytes()}") }
     val path = !effect { createTargetDirectory(source, target) }
-    val paths = !!effect {
+    val validatedPaths = !effect {
       path.ankFiles().map { (index, p) ->
-        suspend {
+        effect(suspend {
```

Comment:
is suspend even needed anymore? `effect {`

Reply:
Thanks!